internal/cmd/stack/stack_selector.go

```diff
@@ -7,18 +7,33 @@ import (
 
 	"github.com/manifoldco/promptui"
 	"github.com/pkg/errors"
+	"github.com/shurcooL/graphql"
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
 	"github.com/urfave/cli/v2"
 )
 
 var errNoStackFound = errors.New("no stack found")
 
+type errStackWithIdNotFound struct {
```

Comment:
I would be for removing this but if you dont:

```suggestion
type errStackWithIDNotFound struct {
```

Reply:
got rid of it in the latest version